WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/PrefMasterSwitchToolbarView.kt

```diff
@@ -149,7 +149,7 @@ class PrefMasterSwitchToolbarView @JvmOverloads constructor(
     }
 
     @Suppress("MemberVisibilityCanBePrivate")
-    fun setToolbarTitleContentDescription(titleContentDescription: String?) {
+    fun setToolbarTitleContentDescription() {
```

Comment:
The `set` prefix is kind of a sacred word in Java now. 😄 It's used for setters but this one is no longer a setter. And it _sounds like a setter_ for `this.titleContentDescription`. I think we can just inline this in `loadInitialState` or rename it to `updateToolbarSwitchForAccessibility()`. What do you think? 🙂

Reply:
Yes, that's a really good point indeed. Let me make that change. Thank you 🙇 I will do the name update so it's better communicates its intention. I would prefer to keep it separate so the `loadInitialState` has a clear responsibility of simply calling to the respective member functions to set up the state. Resolved here 7f26741